libs/api-bundle/src/Util/ControllerReflector.php

```diff
@@ -45,6 +45,8 @@ public function resolveControllerMethod(mixed $controllerSpec): ?ReflectionMetho
             }
 
             try {
+                assert(is_string($controllerClass) || is_object($controllerClass));
```

Comment:
Neni `is_string` reseny uz na radku 39?

Reply:
ne, protože se tam znovu zavolá $controllerClass = $this->container->get($controllerClass);